src/scenes/SelectNextGame/SelectNextGame.tsx

```diff
@@ -23,81 +23,67 @@ import {
   WaitingMessage,
 } from './styles';
 
-enum Visibility {
-  Invisible,
-  Visible,
-}
-
-interface GameCard {
-  id: number;
-  text: string;
-  src: string;
-}
-
 export default function SelectNextGame() {
-  const userData = JSON.parse(window.localStorage.getItem('userData'));
+  const {user, setUser, room, setRoom} = useGlobalContext();
   let nextGame = '';
 
   const navigate = useNavigate();
   const [nextGameName, setNextGameName] = useState('');
-  const [games, updateGames] = useState<GameCard[]>(gameList);
   const [number, setNumber] = useState<number>(-1);
 
-  const [turnVisibility, setTurnVisibility] = useState<Visibility>(
-    Visibility.Invisible
-  );
-  const [ownerVisibility, setOwnerVisibility] = useState<Visibility>(
-    Visibility.Invisible
-  );
   const [rouletteIsSpinning, setRouletteIsSpinning] = useState<boolean>(false);
   const [currentPlayer, setCurrentPlayer] = useState<string>();
 
   //SOCKET///////////////////////////////////////////////////////////////////////////////////////
 
   const socket = SocketConnection.getInstance();
-  let isMyTurn = false;
 
   useEffect(() => {
-    socket.addEventListener('player-turn', (turnID) => {
-      socket.push('get-player-name-by-id', turnID);
-      if (turnID === socket.socket.id) {
-        setTurnVisibility(Visibility.Visible);
-        isMyTurn = true;
-      }
-    });
-    socket.push('player-turn', userData.roomCode);
-
-    socket.addEventListener('room-owner-is', (ownerID) => {
-      console.log();
-      if (ownerID === socket.socket.id) {
-        setOwnerVisibility(Visibility.Visible);
-      }
+    socket.addEventListener('lobby-update', (reply) => { 
+      const newPlayerList = JSON.parse(reply);                  //newPlayerList = Player[]
```

Comment:
Esse aqui é um comentário para dizer o tipo de newPlayerList ou um comando que não é utilizado mais? Se for pra dizer o tipo, faz igual um outro comentário que vc deixou com : ao invés de =